### DISCUSSION TITLE
Support request: Fig was better no recommendations are coming

### DISCUSSION THREAD

**Discussion by SHVM-09** [aws/codewhisperer-command-line-discussions#91] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

need recommendations

### Actual behaviour

no recommendations

**Comment by grant0417** [comment]
Can you give some more details here or post a video of what is happening? Are you not getting any autocompletes?

**Comment by brendanfalk** [comment]
Hey! Just wanted to check in here. Could you please run `cw issue` to create a new ticket? 

We'd love to help but hard for us to debug without a video or any other context.
